usdt-impl/src/no-linker.rs

```diff
@@ -104,6 +112,7 @@ fn compile_probe(
     let out = quote! {
         #[allow(unused)]
         macro_rules! #macro_name {
+            #no_args
```

Comment:
I might rename this to "#no_args_branch` or `#no_args_branch` or something, to indicate it's really a (possibly-empty) separate matcher/transcriber arm of the macro.